### ISSUE TITLE
Lat / Long missing South / North distinction

### ISSUE THREAD

**Issue by BeriBGood** [Martenz/trackanalysis#2]
Hello,
I found an error in the parsing of the IGC file:

`    //collecting all points
    if (item.substring(0,1)==="B"){
      btime = new Date( year,month,day, parseInt(item.substring(1,3)),parseInt(item.substring(3,5)),parseInt(item.substring(5,7)) );
      blon = parseInt(item.substring(7,9))+(parseInt(item.substring(9,11))+parseInt(item.substring(11,14))/1000.0)/60.0; //degrees and ,minutes only (eg 4555321 = 45 degrees 55.321 minutes)
      blat = parseInt(item.substring(15,18))+(parseInt(item.substring(18,20))+parseInt(item.substring(20,23))/1000.0)/60.0; //degrees and ,minutes only (eg 4555321 = 45 degrees 55.321 minutes)
      //check if it is E or W and manage it
      if (item.substring(23,24)==="W"){
        blon = -blon;
};`

This should be:

`    //collecting all points
    if (item.substring(0,1)==="B"){
      btime = new Date( year,month,day, parseInt(item.substring(1,3)),parseInt(item.substring(3,5)),parseInt(item.substring(5,7)) );
      blon = parseInt(item.substring(7,9))+(parseInt(item.substring(9,11))+parseInt(item.substring(11,14))/1000.0)/60.0; //degrees and ,minutes only (eg 4555321 = 45 degrees 55.321 minutes)
      blat = parseInt(item.substring(15,18))+(parseInt(item.substring(18,20))+parseInt(item.substring(20,23))/1000.0)/60.0; //degrees and ,minutes only (eg 4555321 = 45 degrees 55.321 minutes)
      //check if it is N or S and manage it
      if (item.substring(14,15)==="S"){
        blong = -blong;
      //check if it is E or W and manage it
      if (item.substring(23,24)==="W"){
        blat = -blat;
};`

Sorry for the format, the styling is not working.
Thanks a lot for this js, I will try to use some of it for my own flightbook.

**Comment by Martenz** [comment]
Sorry just read your comment, thanks to you for the comment, have you done something then with this?

Best regards,

M.

**Comment by BeriBGood** [comment]
Yes I did, something pretty "ugly", this is a messy coding, but it works.
I think about releasing the code in github as soon as I find time to clean it.
